### PR TITLE
Add Introduction tutorial

### PR DESCRIPTION
**Pull request by tenmaster**
Added:
Introduction tutorial
Modified:
index.yml

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/android/00-introduction.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Introduction
+description: Short Introduction to the Auth0 Android Quickstarts.
+---
+
+<%= include('../../_includes/_github', { link: 'https://github.com/inaka/auth0-android-sample/tree/00-introduction', }) %>
+
+This is a short introduction to the Auth0 Android Quickstarts.
```
Comment (mpaktiti) [inline articles/native-platforms/android/00-introduction.md]: @tenmaster Replace this text with the one from login: "This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your android apps with Auth0."

File: articles/native-platforms/android/00-introduction.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+#### 1. Create an application
+
+<%= include('../../_includes/_new_app') %>_
+
+![App Dashboard](/media/articles/angularjs/app_dashboard.png)
+
+#### 2. Configure your callback URLs
+
+<%= include('../_includes/_callback-url-introduction') %>
```
Comment (mpaktiti) [inline articles/native-platforms/android/00-introduction.md]: @tenmaster The page does not display and I think that's the reason. I cannot find directory `https://github.com/auth0/docs/tree/master/articles/native-platforms/_includes`

I can see this under `client-platforms`: https://github.com/auth0/docs/blob/e213af18660b059cc09f0a067f7ecb47bfd0c158/articles/client-platforms/_includes/_callback-url-introduction.md

File: articles/native-platforms/android/00-introduction.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+* Emulator - Nexus5X - Android 6.0 
+  :::
+  
+## Sample Projects
+
+Each tutorial in the serie includes a link to its corresponding sample project, which expose how to achieve the goal from the tutorial in question. You can check out all the samples [here](https://github.com/auth0-samples/auth0-android-sample).
+
+## Dependencies
+
+Each tutorial will require you to use either [Lock](https://github.com/auth0/Lock.Android) or the [Auth0.swift](https://github.com/auth0/Auth0.Android) library.
```
Comment (volbap) [inline articles/native-platforms/android/00-introduction.md]: ... , or both?

(at least on iOS most of samples required both of them)

Comment (tenmaster) [inline articles/native-platforms/android/00-introduction.md]: @pablolvillar not in android as one includes the other.

**Pull request closed without merging**